helpers/hydration-proposal.js

```diff
@@ -35,13 +35,28 @@ async function generateProposal(
   registerAssets = [],
   whitelist = false,
   newFeePaymentAssets = [],
-  dispatchAsSell = []
+  dispatchAsSell = [],
+  createPoolsWithPegs = [],
```

Comment:
instead of adding these forever for any usecase please add them to batch normally

 e.g generate tx to create pool in proposal and then just add it to batch at right time
 
 you can use api from this file for simplicity, you dont have to create your own interface

Reply:
this function should be more generic, i just did it as i was lazy to expose pjs api, but i see now that i was mistake. i didnt want to establish this wrong pattern